src/zenml/utils/dashboard_utils.py

```diff
@@ -42,7 +42,7 @@ def is_cloud_server(server_info: ServerModel) -> bool:
     """
     return (
         "organization_id" in server_info.metadata
-        and "cloud.zenml.io" in server_info.server_url
+        and "cloud.zenml.io" in server_info.dashboard_url
```

Comment:
Just out of curiosity, why did we have to change this?

Reply:
This PR (https://github.com/zenml-io/zenml/pull/2722) essentially switched the URLs of these fields, which means in the latest release `is_cloud_server` is always `False` even for cloud tenants.